casr/src/bin/casr-dojo.rs

```diff
@@ -366,7 +370,11 @@ impl DefectDojoClient {
             debug!("Uploaded crash seed for finding '{}' with id={}", title, id);
         }
 
-        Ok(())
+        Ok(FindingInfo {
```

Comment:
You may just return JSON stored to `finding` variable. Thus you won't need declaring an additional structure. Id can be stored in `finding["id"]`.